### DISCUSSION TITLE
Modeling turbulent jets inside the domain using Vent

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13791] [Q&A]
Dear All, 
 I saw the turbulent jet validation tutorial case that uses synthetic turbulence BC with legth scales and RMS fluctuations . The vent is kept at the outer boundaries of the domain . If we keep  the vent inside by specifying one surface of the release obstacle ex: BOX, can we use  the same turbulent Jet BC's ( Length scale and RMS  as an input ) , Pls clarify ?

**Comment by drjfloyd** [comment]
You use synthetic turbulence for VENTs inside the domain.

For these kinds of questions it can often be faster to get an answer to just create a simple test case.  It is also a good way to become more procificent with using FDS.

**Reply by Muthu-V** [reply]
Thank you for confirming it. I saw it is working already . Just wanted to confirm with the developers to be sure , if there any additional thing that I need to consider or care about when we release it inside the domain  using Vent BC with synthetic turbulent inlets for turbulent jets. Used slip walls for the other surfaces of the OBST. Would be nice to have a conical obstruction so that jets can be released on the circular surface and the rest can be set as slip walls.
